contracts/InterchainTokenService.sol

```diff
@@ -106,14 +107,17 @@ contract InterchainTokenService is
         address gasService_,
         address interchainTokenFactory_,
         string memory chainName_,
-        address[] memory tokenManagerImplementations
+        address tokenManager_,
```

Comment:
```suggestion
        address tokenManagerImplementation,
```
Maybe?

Reply:
done but not renaming the actual internal variable because it collides with the `tokenManagerImplementation(uint256 tokenManagerType)` function